<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObdScan</class>
 <widget class="QMainWindow" name="ObdScan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="labelVoltage">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>0 V</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushClear">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushExit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="0">
       <widget class="QLabel" name="labelEngineDisplacement">
        <property name="text">
         <string>Displacement:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelLoad">
        <property name="text">
         <string>0 %</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelCoolant">
        <property name="text">
         <string>0 CÂ°</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelSpeed">
        <property name="text">
         <string>0 km/h</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelSpeedTitle">
        <property name="text">
         <string>Speed:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelCoolantTitle">
        <property name="text">
         <string>Coolant Temp:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comboEngineDisplacement">
        <property name="currentText">
         <string> 250</string>
        </property>
        <item>
         <property name="text">
          <string> 250</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 650</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 1000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 1500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 1600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 2000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 2500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 2700</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 2800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 3000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 3100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 3500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 4000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 4500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 5000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 5200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 5600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string> 6000</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelRpmTitle">
        <property name="text">
         <string>Rpm:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelRpm">
        <property name="text">
         <string>0 rpm</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLoadTitle">
        <property name="text">
         <string>Engine Load:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelFuelConsumption">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelFuel100">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelCommand">
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
